conf/modules.config

```
@@ -451,7 +457,10 @@ process {
         ]
         // Have to use a custom `ext` due to deep nested quotes
         ext.tooldict = '{"ampir":"ampir.tsv", "amplify":".tsv", "macrel":".prediction", "neubi":"neubi.fasta", "hmmer_hmmsearch":".txt"}'
-        ext.args =  "--tooldict '${ext.tooldict}' --cutoff ${params.amp_ampcombi_cutoff}"
+        ext.args = { [
+            "--tooldict '${ext.tooldict}'",
+            "--cutoff ${params.amp_ampcombi_cutoff}"
+        ].join(' ').trim() }
```

This lead to an error: the tooldict does not appear in the command as defined. It is a bit tricky to pass a dictionary to python script via command line. Maybe leave it as it was before, because that was working (after a lot of try and error!). I will eliminate the dictionary part in AMPcombi with the next release and then we can adapt this part here.
```suggestion
       ext.args =  "--tooldict '${ext.tooldict}' --cutoff ${params.amp_ampcombi_cutoff}"
```